File: BarCode SDK/C#/QR Code with Website Url/QrCodeUrlTemplate.cs
//*******************************************************************************************//
//                                                                                           //
// Get Your API Key: https://app.pdf.co/signup                                               //
// API Documentation: https://developer.pdf.co/api/index.html                                //
//                                                                                           //
// Note: Replace placeholder values in the code with your API Key                            //
// and file paths (if applicable)                                                            //
//                                                                                           //
//*******************************************************************************************//


using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;

namespace CreateBarCode
{
    class QrCodeUrlTemplate
    {
        #region Constructors

        public QrCodeUrlTemplate() { }
        public QrCodeUrlTemplate(string Url) { this.Url = Url; }

        #endregion

        #region Properties

        public string Url { get; set; }

        #endregion

        #region Overloaded Methods

        public override string ToString()
        {
            if(string.IsNullOrEmpty(Url))
            return base.ToString();

            return Url;
        }

        #endregion
    }
}

File: BarCode SDK/VB.NET/QR Code with Website Url/QrCodeUrlTemplate.vb
'*******************************************************************************************'
'                                                                                           '
' Get API Key https://app.pdf.co/signup                                                     '
' API Documentation: https://developer.pdf.co/api/index.html                                '
'                                                                                           '
' Note: Replace placeholder values in the code with your API Key                            '
' and file paths (if applicable)                                                            '
'                                                                      '
'*******************************************************************************************'


Public Class QrCodeUrlTemplate

#Region "Constructors"

    Public Sub New()
    End Sub

    Public Sub New(ByVal Url As String)
        Me.Url = Url
    End Sub

#End Region


#Region "Properties"

    Public Property Url As String

#End Region

#Region "Overloaded Methods"

    Public Overrides Function ToString() As String

        If String.IsNullOrEmpty(Url) Then Return MyBase.ToString()

        Return Url

    End Function

#End Region


End Class
